### ISSUE TITLE
Simple View: Custom skin reverts back to default on reboot/log off

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#444]
**Reported by Ageless on 11 Jan 37861310 00:00 UTC**
As first reported [on the BOINC forums](http://boinc.berkeley.edu/dev/forum_thread.php?id=2266), when installing a custom skin in Simple View in BOINC Manager and then you reboot the computer or log off/log back on, the skin reverts back to the default skin. 

I have reproduced this with both skins given in [this post](http://boinc.berkeley.edu/dev/forum_thread.php?id=2266&nowrap=true#13739), installing it and logging off/logging back on, restarting BOINC Manager. 

BOINC 5.10.28, Windows 2000 SP4.

Steps to reproduce:[Install skin of choice in \BOINC\skins\[[br]([br]]
1.)]
2. Open BOINC Manager, Simple View.[Preferences, choose skin, save preferences. [[br]([br]]
3.)]
4. With BM open, log off or reboot (logging off is easier ;-))[Log back on, open BM[[br]([br]]
5.)]
6. Result is default skin.

**Comment by romw** [comment]
**Commented by Ageless on 22 Oct 37861334 22:13 UTC**
I just tested further. 

Set Space as custom skin, went back to Advanced View, closed BM to system tray and logged off. Logged back on, opened BM, Simple View and had default skin. 
I then tested exiting BM before logging off, when coming back the custom skin had now stuck. 

Then logging off with BM open, logging back on keeps the Space skin stuck. 
So there has to be at least one time between them that you exited the GUI and did a log off/reboot.
